code/datums/outfits/outfit.dm

```diff
@@ -155,6 +158,8 @@
 		H.wear_suit.add_fingerprint(H, 1)
 	if(H.wear_mask)
 		H.wear_mask.add_fingerprint(H, 1)
+	if(H.neck)
+		H.wear_pda.add_fingerprint(H, 1)
```

Comment:
This seems... not right.

Reply:
Fixed before this review was made.
But yeah at the start there was a lot of code copying.

Reply:
Tell me what time travel device you own that let you go back and edit things after a review.
But yeah, glad it's fixed.